cli_tools/common/utils/logging/service/log_entry.go

```diff
@@ -172,6 +172,9 @@ type OutputInfo struct {
 }
 
 func (i *InputParams) updateParams(p *string) {
+	logParamsMutex.Lock()
```

Comment:
Why is this called update params when it's updating project info?

Should p called project?

Reply:
There can be more params updated if necessary. I won't provide multiple functions for that.
Let me adjust the project param name